src/OrleansProviders/Options/MemoryStreamOptions.cs

```diff
@@ -0,0 +1,9 @@
+namespace OrleansProviders.Options
+{
+    public class MemoryStreamOptions
+    {
+        public int MaxEventCount { get; set; } = DefaultMaxEventCount;
+
+        public const int DefaultMaxEventCount = 16384;
```

Comment:
Please add a comment where this number comes from.

Reply:
That number is the previously hardcoded number in the `MemoryStreamQueueGrain`. I am unaware of the driver behind the number. I have otherwise added some standard comments to the new options class.